1.html

```diff
@@ -0,0 +1,228 @@
+			content="https://talks.ourtech.community/static/img/og-img/home.png"
+		/>
+
+		<script defer src="static/js/script.js"></script>
+
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+			rel="stylesheet"
+		/>
+
+		<link rel="stylesheet" href="static/css/style.css" />
+
+		<link
+			rel="shortcut icon"
+			href="static/img/favicon.ico"
+			type="image/x-icon"
+		/>
+
+		<title>OTC Talks #1</title>
+	</head>
+
+	<body>
+		<header>
+			<nav>
+				<div class="burger-menu">
+					<div class="line-1"></div>
+					<div class="line-2"></div>
+					<div class="line-3"></div>
+				</div>
+
+				<ul class="navbar-ul hide-burger-menu burger-menu-dropdown">
+					<li><a href="/">Home</a></li>
+					<li><a href="/cfp">Talks CFP</a></li>
+					<li><a href="/rules">Rules</a></li>
+					<li>
+						<a
+							href="https://ourtech.community/coc"
+							target="_blank"
+							rel="noreferrer"
+							>Code of Conduct</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://t.me/OurTechComm"
+							target="_blank"
+							rel="noreferrer"
+							>Join us on Telegram</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/two-years"
+							target="_blank"
+							rel="noreferrer"
+							>Two Years of OTC</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/team"
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks #1</h1>
+
+		<section id="talks-details">
+
+			<h2>Table of Contents</h2>
+			
+			<ul id="talks-list">
+				<li>
+					<a href="#talk-1"
+						>Supercharge your deployments with WASM, by Rishit
+						Dagli</a
+					>
+				</li>
+			</ul>
+
+			<h2 id="when-where">Details</h2>
+
+			<ul>
+				<li><b>Date</b>: 24th December, 2022</li>
+				<li><b>Timing</b>: 10:30 PM IST</li>
+				<li>
+					<b>Where</b>: Online, held before
+					<a
+						href="https://catchup.ourtech.community/summary/111"
+						target="_blank"
+						rel="noreferrer"
+						>OTC CatchUp #111</a
+					>
+				</li>
+				<li><a href="/rules">Rules</a></li>
+				<li>
+					<a
+						href="https://ourtech.community/coc"
+						target="_blank"
+						rel="noreferrer"
+						>Code of Conduct</a
+					>
+				</li>
+			</ul>
+
+		</section>
+
+		<main id="main-content">
+			<section id="talk-1" class="talk-section">
+				<h2 class="talk-title">Supercharge your deployments with WASM</h2>
+				<p>
+					Speaker:
+					<a
+						href="https://twitter.com/rishit_dagli"
+						target="_blank"
+						rel="noreferrer"
+						>Rishit Dagli</a
+					>
+				</p>
+				<h3 id="about">Abstract</h3>
```

Comment:
I think this needs to be

```html
<h3 id="abstract-1">Abstract</h3>
```

because there can be multiple talks.

Please check for other instances.

Reply:
Got it.